### ISSUE TITLE
Event Missing new Person Events "person.*"

### ISSUE THREAD

**Issue by virgofx** [stripe/stripe-php#627]
Looks like the Event class is missing some of the new event constants - specifically the `person.*` events.

PHP Stripe SDK: Master
PHP Version: 7.2

**Comment by remi-stripe** [comment]
@virgofx Thanks for the heads up. I opened a PR to fix this: https://github.com/stripe/stripe-php/pull/628

**Comment by virgofx** [comment]
@remi-stripe While you're at it could you add the constants for the `gender` field. I believe those are the only other constants.
```
const GENDER_MALE = 'male';
const GENDER_FEMALE = 'female';
```

**Comment by ob-stripe** [comment]
Fixed in 6.31.5.
